resources/js/Shared/Components/LanguageSwitch.vue

```diff
@@ -0,0 +1,42 @@
+<script setup>
+import { router, usePage } from '@inertiajs/vue3'
+import { ref } from 'vue'
+
+const locales = [
+  {
+    name: 'Polski',
+    lang: 'pl',
+    iso: 'pl',
+  },
+  {
+    name: 'English',
+    lang: 'en',
+    iso: 'gb',
+  },
+]
+
+const page = usePage()
+
+const currentLocale = ref(page.props.locale)
+
+function setLocale(locale) {
+  if (currentLocale.value !== locale) {
+    currentLocale.value = locale
+    router.post(`/language/${locale}`)
+  }
+}
```

Comment:
```suggestion
const currentLocale = computed(() => usePage().props.locale)
```
Can be simplified after few changes.

Reply:
Changed